src/autodoc2/config.py

```diff
@@ -485,6 +485,18 @@ class Config:
         },
     )
 
+    index_filename: str = dc.field(
+        default="index.rst",
+        metadata={
+            "help": (
+                "The filename of the index file, "
+                "relative to the output directory (in POSIX format)."
```

Comment:
```suggestion
                "relative to `autodocs2_output_dir`(in POSIX format)."
```

output directorycould be confused with the sphinx output directory for builds